provisioning/Samples/service/EnrollmentGroupSample/EnrollmentGroupSample.csproj

```diff
@@ -2,12 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
     <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Devices.Provisioning.Service" Version="1.17.2" />

```

Comment:
Latest is 1.80.0. Update?

Reply:
> Latest is 1.80.0. Update?

[1.18.0](https://www.nuget.org/packages/Microsoft.Azure.Devices.Provisioning.Service/) ?